0_basics/005/solution/solution.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file solution.cpp
+ * @author Alexandr Voronin
+ * @brief `proxy 2` task solution
+ * @version 0.1
+ * @date 2023-06-12
+ * 
+ * @copyright Copyright (c) 2023
+ * 
+ */
+
+#include <iostream>
+using namespace std;
+
+
+int main() {
+    int N;
+    cin >> N;
+
+    int A[N];
```

Comment:
some compilers does not allow to define static arrays with variable size. Please, rewrite this